src/metax_api/api/rest/base/views/file_view.py

```diff
@@ -92,22 +92,40 @@ def list(self, request, *args, **kwargs):
             self.queryset_search_params['project_identifier__in'] = user_projects
         return super().list(request, *args, **kwargs)
 
+    def update(self, request, *args, **kwargs):
+        #This have to be checked before updating common info
+        if not isinstance(self.request.data, dict):
+            raise Http400('request.data is not a json')
```

Comment:
A more general error message would be better, instead of referring directly to django data structures. Something like "request message body must be a single json object".